app/src/main/java/nl/tudelft/b_b_w/model/BlockFactory.java

```diff
@@ -31,4 +31,9 @@ public static Block getBlock(String type, String _owner, String _ownHash, String
                 throw new IllegalArgumentException("Invalid type of block: " + type);
         }
     }
+
+    public static Block getBlock(User owner, User user, String ownHash, String previousHashChain, String previousHashSender, String publicKey, boolean revoke) {
```

Comment:
The new getBlock is added with eye on future usability. The current methods require too much parameters, and while this method still requires a lot, it tries to make it more clear what is meant since it uses the new user class instead of all string parameters. Javadoc will be added shortly.